TESTS/mbed_hal/stack_size_unification/main.cpp

```diff
@@ -31,19 +31,11 @@ extern osThreadAttr_t _main_thread_attr;
 #endif
 extern uint32_t mbed_stack_isr_size;
 
-#if !defined(MBED_CONF_RTOS_PRESENT)
-#define EXPECTED_ISR_STACK_SIZE                  (4096)
-#else
-#define EXPECTED_ISR_STACK_SIZE                  (1024)
-#endif
+#define EXPECTED_ISR_STACK_SIZE                  (MBED_CONF_TARGET_BOOT_STACK_SIZE)
 
-#if defined(TARGET_NUCLEO_F070RB) || defined(TARGET_STM32F072RB) || defined(TARGET_TMPM46B) || defined(TARGET_TMPM066)
```

Comment:
All of NUCLEO_F070RB, STM32F072RB and TMPM46B have thread stack sizes overriden in rtos/source/TARGET_CORTEX/mbed_lib.json, so we don't need target-dependent code here. As for TARGET_TMPM066, it's not supported anymore).